src/ameedf/jb8223/assignments/hw02/ClockBonus2.java

```diff
@@ -1,7 +1,187 @@
 package ameedf.jb8223.assignments.hw02;
 
-/**
- * The same clock where minimum memory is occupied
- */
+
 public class ClockBonus2 {
+	// Make variables
+		private int hours;
+		private int minutes;
+		private int seconds;
+		private int milliseconds;
+		private boolean twentyFourDisplay;  
```

Comment:
do you really need it ? it's a parameter you get each time from "outside" the class